swagger.yaml

```diff
@@ -0,0 +1,445 @@
+swagger: "2.0"
+info:
+  description: "This is the Team 1 group project for Boston University's MET CS 673 class in Summer 1 2022.  The application is modeled after personal finance applications like Mint and Truebill and allows users to input bank account and transaction data.  This data can then be accessed, modified, or voided at the user's discretion and it allows them to see their spending habits at a high level. \n\n Team members: Jean Dorancy, Timothy Flucker, Xiaobing Hou, Weijie Liang"
+  version: "1.0.0"
+  title: "Trackr - Personal Transaction Management"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "tflucker@bu.edu"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "trackr.swagger.io"
+basePath: "/v2"
+tags:
+  - name: "User Management"
+  - name: "Bank Account Management"
+  - name: "Transaction Management"
+schemes:
+  - "https"
+  - "http"
+paths:
+  /user:
+    post:
+      tags:
+        - "User Management"
+      summary: "Add a new user to the application"
+      description: "This API will allow a person to create a new user in the application by specifying a username and a password. If the username is unique, a new user will be created and a success message will be returned.  If the username is not unique, no user will be created and an error message will be returned."
+      operationId: "addUser"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - in: "body"
+          name: "body"
+          description: "JSON object used to create a new user in the application."
+          required: true
+          schema:
+            type: "object"
+            properties:
+              username:
+                type: "string"
+                example: "myUser"
+              password:
+                type: "string"
+                format: "password"
+                example: "myPassword"
+              email:
+                type: "string"
+                example: "myUser@email.com"
+      responses:
+        "200": 
+          description: "Successful reponse"
+          schema:
+            $ref: "#/definitions/MessageResponse"
+        "400":
+          description: "Invalid Response, username already exists."
+          schema:
+            $ref: "#/definitions/MessageResponse"
+  /bankAccount:
```

Comment:
Can we name paths with hyphens instead of camel case?
Example: `bank-account` instead of `bankAccount`